src/CoreWCF.Primitives/src/CoreWCF.Primitives.props

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+  <ItemGroup>
+    <!-- Exposes EnableCoreWCFOperationInvokerGenerator at build time -->
+    <CompilerVisibleProperty Include="EnableCoreWCFOperationInvokerGenerator" />
```

Comment:
>CompilerVisibleProperty [](http://example.com/codeflow?start=5&length=23)

We're generating C# code. What happens to both this new implementation and the existing injection generator when using with a vb project?